lib/typescript/httpclient/index.ts

```diff
@@ -237,6 +239,15 @@ export class HttpClient {
       return error;
     }
   }
+
+  public async sendMessages(requestPayload: SendMessagesRequestPayload) {
+    try {
+      const response: MessagePayload = await this.doFetchFromBackend('messages.send', requestPayload);
+      return messageMapper(response);
+    } catch (error) {
```

Comment:
type?

Reply:
MessagePayload is a `type annotation` for the shape of the response. I had assume we needed it there.

Reply:
I mean the Error @ogbeche77